### ISSUE TITLE
Hello, Need Any Help

### ISSUE THREAD

**Issue by samuraininja360** [Ekmand/ekmand#1]
Hello Ekmand I see you don't have anything on your bio do you want assistance with your programming?

I know HTML/CSS/JS, Python, Java and C#

**Comment by Ekmand** [comment]
Yeah if you wanna help me make something cool for my bio that'd be awesome ;)

**Comment by samuraininja360** [comment]
All right!
